bundle/Controller/ContentEditController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryFormsBundle\Controller;
+
+use eZ\Bundle\EzPublishCoreBundle\Controller;
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\API\Repository\LocationService;
+use EzSystems\RepositoryForms\Form\ActionDispatcher\ActionDispatcherInterface;
+use EzSystems\RepositoryForms\Form\Type\Content\ContentCreateType;
+use EzSystems\RepositoryForms\View\ContentEdit\ContentEditSuccessView;
+use EzSystems\RepositoryForms\View\ContentEdit\ContentEditView;
+use Symfony\Component\HttpFoundation\Request;
+
+class ContentEditController extends Controller
+{
+    /**
+     * @var ContentTypeService
+     */
+    private $contentTypeService;
+
+    /**
+     * @var ContentService
+     */
+    private $contentService;
+
+    /**
+     * @var LocationService
+     */
+    private $locationService;
+
+    /**
+     * @var ActionDispatcherInterface
+     */
+    private $contentActionDispatcher;
+
+    public function __construct(
+        ContentTypeService $contentTypeService,
+        ContentService $contentService,
+        LocationService $locationService,
+        ActionDispatcherInterface $contentActionDispatcher
+    ) {
+        $this->contentTypeService = $contentTypeService;
+        $this->locationService = $locationService;
+        $this->contentActionDispatcher = $contentActionDispatcher;
+        $this->contentService = $contentService;
+    }
+
+    public function createWithoutDraftAction(ContentEditView $view)
+    {
+        if ($view->getTemplateIdentifier() === null) {
+            $view->setTemplateIdentifier('EzSystemsRepositoryFormsBundle:Content:content_edit.html.twig');
```

Comment:
This isn't meant to remain, I'll add that to the description.

If you dig a bit, _it is personalizable`. This particular template is the default one, and it will be removed from there ultimately. The whole thing is processed by the view API, and therefore connected to the override system. With this config, every view should already use this template by default, and it should be possible to change it already.

I just haven't reached the point where you have semantic config for that, but you can get a piece of the idea here: https://github.com/ezsystems/repository-forms/pull/67/files#diff-1b4b073bac6c1d6bfbce74d7a9a7a115R2.

I think I'll keep the override part for a further PR, given that this part already does something. One of the next steps is to allow custom templates/controllers for `content_edit` by means of semantic config.